kartograf/util/generate_data.py

```diff
@@ -0,0 +1,105 @@
+"""
+A module to generate IP networks and ASNs for testing purposes.
+"""
+
+import ipaddress
+from random import randint
+
+MAX_ASN = 33521664
+
+def generate_ip(ip_type="v4", subnet_size="16"):
```

Comment:
How about having these test specific util functions under a util folder in tests, similar to bitcoin core: https://github.com/bitcoin/bitcoin/tree/master/test? If we don't need these in the normal code, I would just put them in the test space and let them be shared there.